sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Statistics.scala

```diff
@@ -32,5 +32,11 @@ package org.apache.spark.sql.catalyst.plans.logical
  * @param sizeInBytes Physical size in bytes. For leaf operators this defaults to 1, otherwise it
  *                    defaults to the product of children's `sizeInBytes`.
  * @param isBroadcastable If true, output is small enough to be used in a broadcast join.
+ * @param estimatedSize Estimated size in bytes.
+ * @param rowCount Estimated number of rows.
  */
-case class Statistics(sizeInBytes: BigInt, isBroadcastable: Boolean = false)
+case class Statistics(
+    sizeInBytes: BigInt,
+    isBroadcastable: Boolean = false,
+    var estimatedSize: Option[BigInt] = None,
```

Comment:
I'll make a copy when we set estimatedSize/rowCount, just like how we set isBroadcastable, right?